lua/nvim-treesitter/health.lua

```diff
@@ -94,6 +94,9 @@ local function install_health()
       )
     end
   end
+
+  local systeminfo = vim.fn.has "win32" == 1 and 'systeminfo | findstr /B /C:"OS Name" /C:"OS Version"' or "uname -a"
```

Comment:
this command takes a while to run on Windows. `cmd /c ver` is faster but will show `Windows 10.0.22000` for Windows 11 (we probably just want to know that it is Windows, maybe build version)

Reply:
better to use `vim.loop.os_uname()` https://github.com/neovim/neovim/pull/21157